### ISSUE TITLE
CLI Support

### ISSUE THREAD

**Issue by scs2771** [Connehito/cake-sentry#63]
Is there a way to setup this to send CLI errors to sentry?

**Comment by SparkyRih** [comment]
Doesn't the library already do that? I don't know because I'm not even able to get it to work in the first place (see my issue, which has still not had any replies :S )...

**Comment by o0h** [comment]
We have enabled ConsoleErrorHandler in bootstrap.php, so it is assumed to work in the CLI.

https://github.com/Connehito/cake-sentry/blob/master/config/bootstrap.php#L22
[bootstrap]

[Loading the plugin with bootstrap enabled](https://book.cakephp.org/4/en/plugins.html#plugin-hook-configuration), can you try to see if sending errors when the CLI runs is working?
If it doesn't work, please let us know what version of CakePHP you are using and what version of the plugin you are using.
Also, if possible, please tell me what ErrorHandler is loaded.

**Comment by SparkyRih** [comment]
Why is this guy getting an answer right away, and am I waiting for 2 weeks on issue #62?

**Comment by scs2771** [comment]
I ran a few test and it looks like errors are going to sentry properly. But if there is a Plugin not found exception it is not going to sentry. 

I also noticed that the version of cake-sentry we have on this application is the 1.0 and not the 2.x version. I forgot to check this plugin when we updated the CakePHP to 3.9. We will update this and make sure the config files are correct.

If this does not solve the issue I will let you know.
